src/CommonDBTM.php

```diff
@@ -1763,6 +1763,7 @@ protected function cleanLockeds()
                 )
                 || !isset($this->input['_transfer'])
             )
+            && !isset($this->input['_ignore_locks'])
```

Comment:
It could be used by transfer process, so we could remove specific usage of `_lock_updated_fields` and the complicated check that is using to it.